pyrk/utilities/plotter.py

```diff
@@ -1,10 +1,10 @@
 # Licensed under a 3-clause BSD-style license - see LiCENSE.rst
 import os
 import matplotlib
-matplotlib.use("Agg")
 import matplotlib.pyplot as plt
 import matplotlib.cm as cmx
 import matplotlib.colors as colors
+matplotlib.use("Agg")
```

Comment:
Thank you for the prod -- this should not be moved, or should be before pyplot is imported. I will find the documentation on this and comment that import for the next person.